README.md

```diff
@@ -30,7 +30,17 @@ FluffyChat is an open source, nonprofit and cute matrix messenger app. The app i
 - And much more...
 
 ## Setup
+For macOS or Linux, execute the following command in Terminal.
+```bash
+curl --proto '=https' --tlsv1.2 -sSf https://sh.rustup.rs | sh
+```
+For Windows, you can use the [Rust Installer](https://static.rust-lang.org/rustup/dist/x86_64-pc-windows-msvc/rustup-init.exe).
+
+In case you have Rust already installed, make sure to update it to latest version:
 
+```bash
+rustup update
+```
```

Comment:
Can you explain why we need `Rust`?

Reply:
@sherlockvn  I think you should reference super_clipboard pub url

Reply:
done